.circleci/config.yml

```diff
@@ -41,13 +35,13 @@ references:
 
     install_node_ci: &install_node_ci
       - node/install-packages:
-          cache-version: << pipeline.parameters.node-cache-version >>
+          cache-version: << pipeline.parameters.cache-version >>
           
     install_demisto_sdk: &install_demisto_sdk
```

Comment:
I don't love that this isn't explicit what it's installing in the global environment and is dependent upon where it is executed. If you are going to leave it this way please add a comment explaining the behavior you expect.

Reply:
it's always using the files it needs. npm ci - package-lock.json. pipenv - Pipfile and Pipfile.lock. It's always on the root of the demisto-sdk.